recipes/cargo-make/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
This file should not be needed.

Reply:
On windows it failed to find a version for cargo, commenting that I should use `rustup` or set this file

Reply:
I may iterate a bit if windows fails

Reply:
It seems to work, the file is removed